apps/web/src/components/Router.tsx

```diff
@@ -916,6 +924,17 @@ const createRoute = (
                     </RequireAuth>
                   ),
                 },
+                {
```

Comment:
Should this page be behind the feature flag?

Reply:
Ahh yeah, that makes sense 😆 Fixed in [move page behind feature flag](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9755/commits/4c5e9f3586c59900b31fdf83a9807e3f9d9c1f3d)